apps/e2e/indicators-storybook/src/app/icon/icon.component.html

```diff
@@ -0,0 +1,26 @@
+<sky-icon icon="calendar"></sky-icon>
+<sky-icon icon="calendar" size="lg"></sky-icon>
+<sky-icon icon="calendar" size="2x" iconType="skyux"></sky-icon>
+<sky-icon icon="calendar" iconType="skyux" variant="line" size="3x"></sky-icon>
+<sky-icon
+  data-sky-id="icon-demo"
+  icon="calendar"
+  iconType="skyux"
+  variant="solid"
+  size="4x"
+  [fixedWidth]="true"
+></sky-icon>
+
+<div>
+  <sky-icon icon="bars" size="5x"></sky-icon>
+</div>
+<div>
+  <sky-icon icon="chevron-down" size="5x"></sky-icon>
+</div>
+
+<div>
+  <sky-icon icon="bars" size="5x" [fixedWidth]="true"></sky-icon>
+</div>
+<div>
+  <sky-icon icon="chevron-down" size="5x" [fixedWidth]="true"></sky-icon>
+</div>
```

Comment:
chose calendar icon bc it has both fa and skyux versions, with line and solid variant
showcases all sizes supported
showcases current fa implementation of fixed width

chose not to have visual tests for all available icons and variants due to the lengthy process required to add an icon or variant.